OAIMetadataFormat_JATS.php

```diff
@@ -260,25 +260,38 @@ protected function _mungeMetadata($doc, $journal, $article, $section, $issue) {
 		while ($titleGroupNode->hasChildNodes()) $titleGroupNode->removeChild($titleGroupNode->firstChild);
 		$titleNode = $titleGroupNode->appendChild($doc->createElement('article-title'));
 		$titleNode->setAttribute('xml:lang', substr($article->getLocale(),0,2));
-		$articleTitleText = $doc->createTextNode($article->getTitle($article->getLocale()));
+		$articleTitleText = $doc->createTextNode(
+			$this->mapHtmlTagsForTitle(
+				$article->getCurrentPublication()->getLocalizedTitle(
+					$article->getLocale(), 
+					'html'
+				)
+			)
+		);
 		$titleNode->appendChild($articleTitleText);
-		if (!empty($subtitle = $article->getSubtitle($article->getLocale()))) {
-			$subtitleText = $doc->createTextNode($subtitle);
+		if (!empty($subtitle = $article->getCurrentPublication()->getLocalizedSubTitle($article->getLocale(), 'html'))) {
+			$subtitleText = $doc->createTextNode($this->mapHtmlTagsForTitle($subtitle));
 			$subtitleNode = $titleGroupNode->appendChild($doc->createElement('subtitle'));
 			$subtitleNode->setAttribute('xml:lang', substr($article->getLocale(),0,2));
 			$subtitleNode->appendChild($subtitleText);
 		}
-		foreach ($article->getTitle(null) as $locale => $title) {
-			if ($locale == $article->getLocale()) continue;
-			if (trim($title) === '') continue;
+		foreach ($article->getCurrentPublication()->getTitles(null, 'html') as $locale => $title) {
+			if ($locale == $article->getLocale()) {
+				continue;
+			}
+			
+			if ( trim($title = $this->mapHtmlTagsForTitle($title)) === '' ) {
+				continue;
+			}
+			
 			$transTitleGroupNode = $titleGroupNode->appendChild($doc->createElement('trans-title-group'));
 			$transTitleGroupNode->setAttribute('xml:lang', substr($locale,0,2));
 			$titleNode = $transTitleGroupNode->appendChild($doc->createElement('trans-title'));
 			$titleText = $doc->createTextNode($title);
 			$titleNode->appendChild($titleText);
-			if (!empty($subtitle = $article->getSubtitle($locale))) {
+			if (!empty($subtitle = $article->getCurrentPublication()->getLocalizedSubTitle($locale, 'html'))) {
 				$subtitleNode = $transTitleGroupNode->appendChild($doc->createElement('trans-subtitle'));
-				$subtitleText = $doc->createTextNode($subtitle);
+				$subtitleText = $doc->createTextNode($this->mapHtmlTagsForTitle($subtitle));
```

Comment:
I think this probably won't work; `createTextNode` will escape any HTML elements.

Reply:
@asmecher Fixed it . This is how ti looked which I am not sure if correct or not 

<img width="536" alt="CleanShot 2023-02-17 at 01 26 17@2x" src="https://user-images.githubusercontent.com/2587979/219467348-fb62cbd6-70b4-4ef5-a7ca-6d986e36107d.png">

XML extract from page source 

```xml
<?xml version="1.0" encoding="UTF-8"?>
<?xml-stylesheet type="text/xsl" href="http://ojs-main.test/lib/pkp/xml/oai2.xsl" ?>
<OAI-PMH xmlns="http://www.openarchives.org/OAI/2.0/"
	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
	xsi:schemaLocation="http://www.openarchives.org/OAI/2.0/
		http://www.openarchives.org/OAI/2.0/OAI-PMH.xsd">
	<responseDate>2023-02-16T19:23:21Z</responseDate>
	<request verb="GetRecord" metadataPrefix="jats" identifier="oai:ojs2.ojs-main.test:article/37">http://ojs-main.test/index.php/publicknowledge/oai</request>
	<GetRecord>
		<record>
			<header>
				<identifier>oai:ojs2.ojs-main.test:article/37</identifier>
				<datestamp>2023-02-16T19:23:21Z</datestamp>
				<setSpec>publicknowledge:ART</setSpec>
			</header>
			<metadata>
<article xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="https://jats.nlm.nih.gov/publishing/1.1/" xml:lang="en" dtd-version="1.1" specific-use="eps-0.1">
			<front>
			<journal-meta>
				<journal-id journal-id-type="publisher">publicknowledge</journal-id><journal-id journal-id-type="ojs">publicknowledge</journal-id>
				<journal-title-group>
			<journal-title xml:lang="en">Journal of Public Knowledge</journal-title><trans-title-group xml:lang="fr"><trans-title>Journal de la connaissance du public</trans-title></trans-title-group>
</journal-title-group>			<contrib-group><contrib contrib-type="jmanager"><name><surname>admin</surname><given-names>admin</given-names></name></contrib><contrib contrib-type="jmanager"><name><surname>Vaca</surname><given-names>Ramiro</given-names></name></contrib><contrib contrib-type="jmanager"><name><given-names>Journal Manager 1</given-names></name></contrib><contrib contrib-type="editor"><name><surname>Barnes</surname><given-names>Daniel</given-names></name></contrib><contrib contrib-type="editor"><name><given-names>Editor 1</given-names></name></contrib><contrib contrib-type="editor"><name><given-names>Editor 2</given-names></name></contrib><contrib contrib-type="secteditor"><name><surname>Buskins</surname><given-names>David</given-names></name></contrib><contrib contrib-type="secteditor"><name><surname>Berardo</surname><given-names>Stephanie</given-names></name></contrib><contrib contrib-type="secteditor"><name><surname>Inoue</surname><given-names>Minoti</given-names></name></contrib></contrib-group><issn pub-type="epub">0378-5955</issn>			<issn pub-type="ppub">0378-5955</issn>			<publisher><publisher-name>Public Knowledge Project</publisher-name></publisher>
		</journal-meta>
		<article-meta>
			<article-id pub-id-type="publisher-id">37</article-id>
			<article-categories><subj-group subj-group-type="heading" xml:lang="en"><subject>Articles</subject></subj-group></article-categories>
			<title-group><article-title xml:lang="en">New Test <bold>Bold</bold> <italic>Italic</italic> <underline>Underline</underline> <sup>Superscript</sup> <sub>SubScript</sub> _______ 000001</article-title><subtitle xml:lang="en">Some <italic><bold>new</bold></italic> <underline>sub</underline> <italic><sup>title</sup> <underline><bold><sub>addition</sub></bold></underline></italic> _______ 0000001</subtitle></title-group>
			<contrib-group content-type="author">
				<contrib>
					<name name-style="western">
						<given-names>Author</given-names>
					</name>
					<email>author@mail.test</email>
				</contrib>
			</contrib-group>
			<pub-date date-type="collection"><year>2014</year></pub-date><pub-date date-type="pub" publication-format="epub"><day>14</day><month>02</month><year>2023</year></pub-date>
			<volume seq="2">1</volume><issue>2</issue><issue-id>1</issue-id><permissions>
				<copyright-statement>Copyright (c) 2022 Journal of Public Knowledge</copyright-statement>
				<copyright-year>2022</copyright-year>
				<copyright-holder>Journal of Public Knowledge</copyright-holder>
			</permissions>
			
		<self-uri xlink:href="[http://ojs-main.test/index.php/publicknowledge/article/view](view-source:http://ojs-main.test/index.php/publicknowledge/article/view)"/><abstract><p>None</p></abstract></article-meta>
	</front>
</article>			</metadata>
		</record>
	</GetRecord>
</OAI-PMH>
```